cpp/perspective/src/cpp/computed_expression.cpp

```diff
@@ -192,10 +230,16 @@ t_computed_expression::compute(std::shared_ptr<t_data_table> source_table,
     std::shared_ptr<t_vocab> vocab) const {
     // TODO: share symtables across pre/re/compute
     exprtk::symbol_table<t_tscalar> sym_table;
+
+    // pi, infinity, etc.
     sym_table.add_constants();
 
-    REGISTER_COMPUTE_FUNCTIONS(vocab)
+    // "True" and "False" as boolean scalars
+    REGISTER_SCALAR_CONSTANTS()
 
+    // Custom functions from computed_functions.cpp
+    REGISTER_COMPUTE_FUNCTIONS(vocab)
```

Comment:
Do these need to be macros?  They impair readability here.

At minimum, they should take `sym_table` as argument and not name-capture as they do now.